api/rest/v2/index.md

```diff
@@ -216,7 +232,7 @@ For example, the request:
 
 ```bash
 curl --request GET \
-  'https://api.datacommons.org/v2/node?key=AIzaSyCTI4Xz-UW_G2Q2RfknhcfdAnTHq5X5XuI&nodes=geoId/06&property=<-*'
+  'https://api.datacommons.org/v2/node?key=AIzaSyCTI4Xz-UW_G2Q2RfknhcfdAnTHq5X5XuI&nodes=geoId%2F06&property=%3C-%'
```

Comment:
same comment about the `*` encoding getting cut off

Reply:
Hmm, not sure how this happened